scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1311,7 +1319,8 @@ Else
     $BypassInstallationFlag = $null
 }
 
-$retCode = RunTest
+# Evaluate collection of test results for final pass/fail result
+RunTest | ForEach-Object {$retCode = 0} {$retCode = $retCode -bor $_}
 Write-Host "Exit test with code $retCode"
 
 Exit $retCode -gt 0
```

Comment:
@philipktlin @damonbarry is there a particular reason that there's a -gt comparison here? It seems like this will exit with "1" if all the tests pass, since a "0 -gt 0" returns false. Is that by design?

Reply:
Oh, guessing this was because of how RunTest was previously? Is it safe to change to "Exit $retCode"?